src/graphql/types/Mutation/createChat.ts

```
@@ -3,204 +1,151 @@
+      if (found.length !== userIds.length) {
+        throw new TalawaGraphQLError({
+          extensions: {
+            code: "invalid_arguments",
+            issues: [{ argumentPath: ["input", "userIds"], message: "One or more userIds do not exist." }],
+          },
+        });
+      }
+
+    
+      const keyName = isGroup
+        ? name!.trim()
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)